cardano-cli/src/Cardano/CLI/EraBased/Options/Common.hs

```diff
@@ -102,6 +103,27 @@ pNetworkId envCli = asum $ mconcat
     pure <$> maybeToList (envCliNetworkId envCli)
   ]
 
+pTarget :: Maybe (ConwayEraOnwards era) -> Parser (Consensus.Target ChainPoint)
+pTarget Nothing = pure Consensus.VolatileTip
+pTarget (Just ConwayEraOnwardsConway) =
```

Comment:
Casing function has an advantage over pattern matching on particular constructors that you have post-conway eras support out of the box. Using `CardanoEra` would reduce duplication, because you won't have to convert `CardanoEra` to `ConwayEraOnwards` before every usage of this parser.

```suggestion
pTarget :: CardanoEra era -> Parser (Consensus.Target ChainPoint)
pTarget = caseShelleyToBabbageOrConwayEraOnwards (pure Consensus.VolatileTip) $
```

Reply:
Yes, I was thinking I would have to pass `era` in many places, but it turns out it is only in one place, so it is much cleaner. I have addressed it in 8f50384. About the matching on post Conway eras, I just put an underscore in the pattern match, I understand it must be equivalent.